R/DataSet.R

```diff
@@ -884,15 +884,27 @@ get_FV_summary.DataSet <- function(ds, runtime, ...) {
   runtime <- sort(as.numeric(unique(c(runtime))))
   RT <- as.numeric(rownames(data))
   idx <- seq_along(RT)
-  
+
+  if (max(RT) < max(runtime)) { #Avoid forgetting stopped runs
+    data2 <- rbind(data, data[nrow(data),])
+    rownames(data2) <- c(rownames(data), max(c(runtime, RT)) + 1)
+    data <- data2
+  }
+
+  data <- apply(data, 2, function(x) { #Remove NA to preserve monotonicity of mean

```

Comment:
What does this part do? Sorry for the stupid question.. I am no long very familiar with the code.

Reply:
This is just some code I added to ensure the monotonicity of data when not all runs have equal length. It usually shouldn't happen, but if there are NA's for whatever reason, this deals with them so they don't mess up the means.